src/requirements/data/majors/comm.ts

```diff
@@ -0,0 +1,157 @@
+    checker: includesWithSubRequirements(
+      ['COMM 2200'],
+      ['COMM 2450'],
+      ['COMM 2760'],
+      ['COMM 2850'],
+    ),
+    operator: 'or',
+    fulfilledBy: 'credits',
+    minCount: 6
+  },
+  {
+    name: 'Focus Area Upper Level',
+    description: 'Students must take six credits (two courses) of coursework within their declared Focus Area at the 3100+ level.',
+    source: 'https://communication.cals.cornell.edu/undergraduate-program/major-requirements/',
+    checker: includesWithSingleRequirement(
+      'COMM 31**',
+      'COMM 32**',
+      'COMM 33**',
+      'COMM 34**',
+      'COMM 35**',
+      'COMM 36**',
+      'COMM 37**',
+      'COMM 38**',
+      'COMM 39**',
+      'COMM 4***',
+      'COMM 5***',
+      'COMM 6***',
+    ),
+    operator: 'or',
+    fulfilledBy: 'credits',
+    minCount: 6
+  },
+  {
+    name: 'Upper-Level COMM',
+    description: 'Students must complete 9 additional credit hours at the 3100+ level.* Electives can come from any of the focus area lists. '
+    + 'A student may elect to fulfill 3 of these credit hours by taking a third focus area introductory course. '
+    + 'A maximum of 3 credits in either 4970 or 4990 (combined) can be counted toward the upper level major requirements. Refer to the Course and Time Roster for the most up-to-date offerings. ',
+    source: 'https://communication.cals.cornell.edu/undergraduate-program/major-requirements/',
+    checker: includesWithSubRequirements(
+      ['COMM 2179'],
+      ['ANTHR 4330', 'COGST 4330', 'STS 4330'],
+      ['COML 2703', 'ENG 2703', 'MUSIC 2703', 'PMA 2703'],
+      ['COMM 2179'],
+      ['COMM 3110'],
+      ['COMM 3150'],
+      ['COMM 3189'],
+      ['COMM 3200'],
+      ['COMM 3210'],
+      ['COMM 3300'],
+      ['COMM 3350'],
+      ['COMM 3400'],
+      ['COMM 3450'],
+      ['COMM 3460'],
+      ['COMM 3560'],
+      ['COMM 3710'],
+      ['COMM 3720'],
+      ['COMM 3760'],
+      ['COMM 3800'],
+      ['COMM 4200'],
+      ['COMM 4201'],
+      ['COMM 4220'],
+      ['COMM 4250'],
+      ['COMM 4260'],
+      ['COMM 4280'],
+      ['COMM 4292'],
+      ['COMM 4300'],
+      ['COMM 4350'],
+      ['COMM 4360'],
+      ['COMM 4380'],
+      ['COMM 4400'],
+      ['COMM 4410'],
+      ['COMM 4450'],
+      ['COMM 4560'],
+      ['COMM 4650'],
+      ['COMM 4660'],
+      ['COMM 4760'],
+      ['COMM 4800'],
+      ['COMM 4860'],
+      ['COMM 4940'],
+      ['COMM 4970'],
+      ['COMM 4990']
+    ),
+    operator: 'or',
+    fulfilledBy: 'credits',
+    minCount: 15
+  },
+  {
+    name: 'Communication Practica',
+    description: 'Three credits in the COMM 30XX range courses',
+    source: 'https://communication.cals.cornell.edu/undergraduate-program/major-requirements/',
+    checker: includesWithSingleRequirement(
+      'COMM 30**',
+    ),
+    operator: 'or',
+    fulfilledBy: 'credits',
+    minCount: 3
+  },
+  {
+    name: 'First-Year Writing Workshop (FWS)',
```

Comment:
Isn't this covered by college requirements? Or is it an additional one

Reply:
That's actually a really interesting point. I believe that in COMM, you only need 1 FWS unlike all other majors in CALS. Yes, it technically is, but it's not accurate for the COMM major. This is one of the few examples where college requirements depend on the major. However, we don't support that yet

Reply:
Then we should get rid of this right? Since the req is already covered in college requirements (just at a more extreme level then it should be). And then make a Monday item in "Requirements Handling" to keep track of that discrepancy?

Reply:
Right but then we would give the wrong impression that you need two FWS courses as a COMM major